ckanext/videoview/theme/templates/video_view.html

```diff
@@ -0,0 +1,5 @@
+<video style="margin:auto; max-width:100%; display:block" controls>
+  <source src="{{ resource.get('video_url') or resource.get('url') }}" type="{{ resource.get('format') }}">
+  <p>Your browser doesn't support HTML5 video.
+  Here is a <a href="{{ resource.get('video_url') or resource.get('url') }}">link to the video</a> instead.</p>
```

Comment:
Make this translatable with something like
```jinja2
{% set video_url=resource.get('video_url') or resource.get('url') %}
<p>
  {% blocktrans %}
    Your browser doesn't support HTML5 video.
    Here is a <a href="{{ video_url }}">link to the video</a> instead.
  {% endblocktrans %}
</p>
```

Reply:
Did it, thanks :-)